plasmapy/__init__.py

```diff
@@ -12,10 +51,6 @@
 
 __name__ = "plasmapy"
 
-__doc__ = ("A community-developed and community-driven open source "
-           "core Python package for plasma physics.")
-
-
 class UnsupportedPythonError(Exception):
```

Comment:
I take exception to this exception!

(not relevant to this pull request at all, but I kept really wanting to say something like that because of #612)